gulpfile.js

```diff
@@ -23,7 +23,7 @@ var srcDir = './src/';
 var outDir = './dist/';
 
 if (argv.verbose) {
-  util.log("Gulp running with options: " + JSON.stringify(argv, null, 2));
+  log("Gulp running with options: " + JSON.stringify(argv, null, 2));
```

Comment:
I would instead remove the `verbose` arg and get rid of the `log` dependency completely, seems a bit useless.

Or maybe `console.log()` is enough if we really want to keep the `verbose` arg.

Reply:
done